### PR TITLE
Feat/match yocto wds config

### PR DESCRIPTION
**Pull request by alexhumphreys**

### CODE REVIEW

File: meta-jaspar/recipes-connectivity/wpa-supplicant/wpa-supplicant/defconfig [hunk 1]
```diff
@@ -516,9 +495,7 @@ CONFIG_CTRL_IFACE_DBUS_NEW=y
 # should be noted that this is mainly aimed at simple cases like
 # WPA2-Personal while more complex configurations like WPA2-Enterprise with an
 # external RADIUS server can be supported with hostapd.
-CONFIG_AP=y
-
-CONFIG_BGSCAN_SIMPLE=y
+#CONFIG_AP=y
```
Comment (alexhumphreys) [inline meta-jaspar/recipes-connectivity/wpa-supplicant/wpa-supplicant/defconfig]: @cajun-rat Not sure, but this might be important...

**Comment by cajun-rat** [comment]
We are going to sit on this until we can confirm that it fixes the WDS issues.  When we can get WDS working with it, we'll merge it but otherwise we are risking introducing bugs.

**Comment by cajun-rat** [comment]
Am I right in thinking that we believe connman 1.31 breaks the streaming use case?

**Comment by alexhumphreys** [comment]
I believe so, not sure what state my tmp/ folder is in but I can rebuild and test later. Maybe it was that CONFIG_AP thing afterall...

**Comment by cajun-rat** [comment]
This seems to stop the Streaming mode working :(

To reproduce:

1) Boot the image
2) Click 'streaming'
3) ssh qemu connmanctl technologies

In a good build (e.g. master) it will say 'Tethering = True' under /net/connman/technology/wifi

With this, it will say 'Tethering = False', and typing 'tether wifi on' in connmanctl will result in an error like so:

```
connmanctl> tether wifi on
Error enabling wifi tethering: Not supported
```

**Comment by stevana** [comment]
Why is this branch needed? It's untested and only seems to bump connman to 1.31 (which we haven't confirmed improves anything).

**Comment by alexhumphreys** [comment]
@stevana it was opened 12 days ago. I think I was trying to match yocto as close as possible to either the vagrant or Vitaly's setup. If we're happy enough with master and if we don't need and/or can't fix connman 1.31, then I guess we don't need this branch.

**Comment by stevana** [comment]
Good, since connman 1.31 already has its own branch; let's close this one.

**Pull request closed without merging**